source/Octopus.Server.Client/Repositories/Async/DeploymentSettingsRepository.cs

```diff
@@ -44,7 +44,7 @@ public async Task<DeploymentSettingsResource> Get(ProjectResource project)
                 throw new NotSupportedException(
                     $"Version Controlled projects are still in Beta. Use {nameof(IDeploymentSettingsBetaRepository)}.");
 
-            await client.Put(deploymentSettings.Link("Self"));
+            await client.Put(deploymentSettings.Link("Self"), project);
```

Comment:
@YuKitsune has a different fix for this on this branch atm: https://github.com/OctopusDeploy/OctopusClients/pull/588/files/72ea3f111ffc59839b0049cabcc83d9530c300d0..e95ebe83e87a0ce87498b90ac7420e5fd2f1d5b3

I think this should have been sending through the deploymentSettings object as part of the PUT?

Reply:
That right, it should be the deploymentSettings, my branch has some other fixes related to wrong links being used too.

Reply:
Have reworked the change, with help from @dylanlerch